### DISCUSSION TITLE
three questions, thanks

### DISCUSSION THREAD

**Discussion by 404gods** [umami-software/umami#2156] [General]
### Describe the feature or enhancement

1. How to view the current version of umami on the dashboard page?
2. Can the dashboard shrink the statistics page? Allow one page to display multiple website information.
3. The language setting cannot be saved, and the default language is still English after re-login.

**Comment by mikecao** [comment]
1. There is no way currently. We will provide a way in the future.
2. I don't know what you mean by this.
3. The language should be saved since it uses local storage.
